packages/ui5-task-nwabap-deployer/lib/ui5-task-nwabap-deployer.js

```diff
@@ -29,6 +29,14 @@ module.exports = async function({ workspace, dependencies, options }) {
     } else {
         options.configuration.connection = Object.assign({}, options.configuration.connection);
     }
+    
+    let sClient = process.env.UI5_TASK_NWABAP_DEPLOYER__CLIENT;
+    
+    if (options.configuration && options.configuration.connection && options.configuration.connection.client) {
+        client = options.configuration.connection.client;
```

Comment:
Variable name is sClient according to your definition (and not client).

Reply:
thanks and nicely spotted. I used the .env file for my testing. I've changed that now